recipes-kernel/kernel-modules/kernel-module-nxp-wlan/mlinux-moal_main-lower-PRINTM_MMSG-log-level-to-KERN_INFO.patch

```diff
@@ -1,4 +1,4 @@
-From b1b218757973fd6a293de5bdbdc75307db07998e Mon Sep 17 00:00:00 2001
+From 9e2200debd7ac4fa8dac4b01949b5bedbc52fe2b Mon Sep 17 00:00:00 2001
 From: Vitor Soares <vitor.soares@toradex.com>
```

Comment:
Please fix the recipe name, because you shouldn't include the version suffix, only the recipe name as a prefix.

Reply:
Sorry for my mistake. Fixed